### ISSUE TITLE
YOLOv3/core/yolov3.py - What does conf_loss consist of

### ISSUE THREAD

**Issue by KaiquanMah** [sniper0110/YOLOv3#2]
Hi Nour,

Just curious. In [YOLOv3/core/yolov3.py](github.com/sniper0110/YOLOv3/blob/main/core/yolov3.py), what does conf_loss in lines 184 to 188 consist of?

`
    conf_loss = conf_focal * (
            respond_bbox * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
            +
            respond_bgd * tf.nn.sigmoid_cross_entropy_with_logits(labels=respond_bbox, logits=conv_raw_conf)
    )
`

Is conf_loss trying to find the 'mismatch' between the ground truth bounding box coordinates and the predicted bounding box coordinates (x_topleft, y_topleft, x_bottomleft, y_bottomleft)?
